src/seedu/addressbook/data/person/Person.java

```diff
@@ -54,7 +57,11 @@ public Address getAddress() {
         return address;
     }
 
-    @Override
+    public int getSequenceNumber() {
```

Comment:
Please fix the indentation and change the tab to spaces in Eclipse, see [here](http://stackoverflow.com/questions/407929/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs)